routing/routing_tests/vehicle_model_test.cpp

```diff
@@ -21,6 +21,9 @@ routing::VehicleModel::InitListT const s_testLimits = {
 
 class TestVehicleModel : public routing::VehicleModel
 {
+  friend void CheckOneWay(initializer_list<uint32_t> const & types, bool expectedValue);
```

Comment:
The methods CheckOneWay and CheckSpeed are declared as friends to be able to use protected methods of the base class: VehicleModel::GetMinTypeSpeed() and VehicleModel::HasOneWayType()